src/geo/transform.js

```diff
@@ -1355,11 +1361,9 @@ class Transform {
 
         this._constraining = true;
 
-        let minY = -90;
-        let maxY = 90;
-        let minX = -180;
-        let maxX = 180;
-        let sy, sx, x2, y2;
+        let minY = 0;
```

Comment:
Removing these initial values since they are never used (and suggest incorrect lat/long units).

Reply:
Should `maxY = -Infinity`?